From: Hiroaki Nakamura <hnakamur@gmail.com>
Date: Tue, 12 Jun 2018 16:32:34 +0900
Subject: Loosen acceptable range for throttling to pass on sbuild

---
 persister/throttle_test.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/persister/throttle_test.go b/persister/throttle_test.go
index 722f3cd..0e78a51 100644
--- a/persister/throttle_test.go
+++ b/persister/throttle_test.go
@@ -35,8 +35,8 @@ func TestThrottleChan(t *testing.T) {
 
 	for _, perSecond := range perSecondTable {
 		bw := doTestThrottleTicker(perSecond)
-		max := float64(perSecond) * 1.05
-		min := float64(perSecond) * 0.95
+		max := float64(perSecond) * 1.5
+		min := float64(perSecond) * 0.5
 		assert.True(t, float64(bw) >= min, fmt.Sprintf("perSecond: %d, bw: %d", perSecond, bw))
 		assert.True(t, float64(bw) <= max, fmt.Sprintf("perSecond: %d, bw: %d", perSecond, bw))
 	}
